src/README.md

```diff
@@ -0,0 +1,23 @@
+Airport Challenge 
+Due Date: 23/01/2023
+Matt Jones
+SE-2301-A
+
+Learning Objectives:
+
+-Be able to build functionality for applications based on user requirements
+-Write some tests using a testing framework to make sure that your code functions correctly
+-Use classes and objects in your code for reproducibility and efficiency
+-Use Git and GitHub to commit work and open a Pull Request
+
+Grading Criteria:
+
+-Have you converted all of the user requirements into a functioning application?
+-Have you tested your code?
+-Does the code follow good practices?
+    -Is the code clean (use appropriate statements, no repeated blocks, short circuit where possible)?
+    -Has the code been developed using OOD? - i.e.
+        -Is the code loosely coupled, highly cohesive and follow the single responsibility principles?
+        -Is domain modelling evidenced in the README?
+Is the code written as simple as possible or is it too complex?
+
```

Comment:
Helped me during the project to save me having to refer to the original instructions but removed at the end.